docs/getting-started.md

```diff
@@ -133,47 +131,46 @@ EXPOSE 80
+* if the hostname is required by the framework/lib then it has to be set to `0.0.0.0`
 
+For our example `npm run prod` is running `PORT=80 node index.js` which satisfies these two requirements
 
 ## Create a Kintoblock on KintoHub from the service we just created
 
-*For more details and screenshots about creating KintoBlocks read more [here](creating-a-kintoblock.md)*
+_For more details and screenshots about creating KintoBlocks read more [here](creating-a-kintoblock.md)_
 
 By this point, you should already have a GitHub repository that is compatible with KintoHub
 
 Next step is connecting your workspace to your GitHub account. Go to edit workspace and click on connect GitHub button.
 
-Now go to create Kintoblock page. Add a Kintoblock name *(name must be unique across all accounts)*, choose an existing repository and select the repository you just created. Set protocol to `HTTP` and language to `Node.js`.
+Now go to create Kintoblock page. Add a Kintoblock name _(name must be unique across all accounts)_, choose an existing repository and select the repository you just created. Set protocol to `HTTP` and language to `Node.js`.
 
-> For the rest of the guide we are going to assume the name is *`nodeexample`*
+> For the rest of the guide we are going to assume the name is _`nodeexample`_
 
-> endpoints are going to be in the following format **http://{microservicename}.api.staging.kintohub.com**
+> endpoints are going to be in the following format **http://api.staging.kintohub.com/{microservicename}**
 
-> Kintoblocks can't be accessed directly, it must be through Applications, if you tried to go to `http://nodeexample.api.staging.kintohub.com/sample/hello` you will get an error.
+> Kintoblocks can't be accessed directly, it must be through Applications, if you tried to go to `http://api.staging.kintohub.com/nodeexample/sample/hello` you will get an error.
 
 ### Tag Latest Commit
 
-*For more details and screenshots about tagging latest commit read more [here](tagging-a-commit.md)*
+_For more details and screenshots about tagging latest commit read more [here](tagging-a-commit.md)_
 
 After creating the Kintoblock, when you are in the Kintoblock manage page, navigate to the branch that you added your code to. And go to the **Commits** section.
 
-Make sure that the last commit you had is in recent commits and wait until it successfully builds. *(you have to refresh the page to get the latest build status update)*.
-
-After it successfully builds click `Tag Latest Commit` this allows you to add a tag to that commit *(similar to how git works)* so you can select that tag in a Application
+Make sure that the last commit you had is in recent commits and wait until it successfully builds. _(you have to refresh the page to get the latest build status update)_.
 
+After it successfully builds click `Tag Latest Commit` this allows you to add a tag to that commit _(similar to how git works)_ so you can select that tag in a Application
```

Comment:
...you can select that particular tag when you are building an Application (before deploying it)